content/articles/lxc-vs-docker-what-is-the-difference-and-why-docker-is-better/index.md

```diff
@@ -0,0 +1,109 @@
+
+### What is the major difference between Linux and Docker containers
+
+LXC focuses on OS containerization, while Docker thrives on application containerization. Docker is single-purpose application virtualization, and LXC is multi-purpose operating system virtualization.
+
+In this case, LXC specializes in deploying Linux Virtual machines. A container is like a VM with a fully functional OS environment. However, the OS has to support and handle the features and capabilities of a Linux environment. You can SSH into an LXC container, operate it like an operating system, and install any application or services, and everything will work as expected.
+
+This is not the case with Docker. Docker specializes in deploying applications. Docker containers aren't lightweight virtual machines. Thus they can't be considered as such. Docker containers are limited to a single application due to their architecture. Although Docker runs natively in a Linux environment, it is not entirely dependent on Linux, and it supports other operating systems such as Windows and macOS.
+
+### Why Docker is better
+
+#### Portability
+
+Probably you have heard a phrase it works on my computer but not theirs.
+
+Consider a case where a company hires a developer to develop an app on oracle WebLogic software. This means the developer has Oracle WebLogic installed on the computer. When the application is fully developed, the developer will share this application with the team, testing the code. The team will then have to repeat the process of installing and configuring the Oracle WebLogic. The same application is shared with the production team, and the cycle repeats itself.
+
+Due to the environmental difference on these three levels, the installation was done separately. This is one of the most comprehensive solutions that Docker container offers. Docker will containerize this single application with all the configuration the application needs to run. The other team only needs to have the Docker demons installed. Then the developer can share this application without having to install additional software regardless of the operating system the other teams are in.
+
+Docker goes hand in hand with versioning. When sharing this application, the version of the services used doesn't change. Thus, the whole application will be shared without breaking the code.
+
+### Flexibility
+
+As we have explained above, Docker knows no environmental boundaries. Linux containers only support containerization with a Linux-based operating system. Thus a container only runs in one environment. That's not the case with Docker. If it works on your computer, it will work on my computer as well. Docker engine is well set to work constantly across all environments. If you set an application on a Windows-based server, it can be easily deployed in a Linux server without compatibility issues.
```

Comment:
```suggestion
As we have explained above, Docker knows no environmental boundaries. Linux containers only support containerization with a Linux-based operating system. Thus, a container only runs in one environment. That's not the case with Docker. If it works on your computer, it will work on my computer as well. Docker engine is well set to work constantly across all environments. If you set an application on a Windows-based server, it can be easily deployed in a Linux server without compatibility issues.
```